src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -136,6 +145,9 @@ def provider_reducer(dict, integration):
 
             if shared_domain:
                 queryset = queryset.filter(email__endswith=shared_domain)
+            else:
+                for filtered_email in filtered_email_domains:
+                    queryset = queryset.exclude(email__endswith=filtered_email)
```

Comment:
`exclude(email__endswith__in)` doesn't work, so i went with this

Reply:
nit: I think you can do `queryset.exclude(email=filtered_email)`

Reply:
i need to check if the person's email _contains / ends with_ the domain though, not that it equals the domain

Reply:
oh you're right, too bad you can't do this in one go